src/main/java/org/terasology/climateConditions/HypothermiaSystem.java

```diff
@@ -48,13 +52,21 @@ public void observeDangerZone(MovedEvent event, EntityRef player) {
         }
     }
 
+    /**
+     * Modifies the walking/running speed of the player.
+     * Is only active iff the player has a {@link HypothermiaComponent}.
+     */
     @ReceiveEvent(components = {HypothermiaComponent.class})
     public void modifySpeed(GetMaxSpeedEvent event, EntityRef player) {
         if (event.getMovementMode() == MovementMode.WALKING) {
             event.multiply(walkSpeedMultiplier);
         }
     }
 
+    /**
+     * Modifies the jump speed of the player.
```

Comment:
Same as above:  `modifyJumpSpeed` modifies the jump speed - no surprises there. But in what way? Another clue here might be the fact that we are not modifying any physical ability of the character but reacting to the respective events.